Eastern-Research-Group/expert-query
# Feature/remove stream national downloads code

## Related Issues:
* None

## Main Changes:
* Removed the stream national downloads logic from the etl.
  * I don't think we will be needing this code for this version, so I figured I would go ahead and remove it. We can reference this PR if we need this for a feature in a future version.

## Steps To Test:
1. Run `npm ci` from the etl folder
2. If necessary, remove the following environment variables from your `etl/.env.local` file: `STREAM_BATCH_SIZE`, `STREAM_HIGH_WATER_MARK`, `STREAM_SERIALLY`, `STREAM_SHARED`
3. Run the etl process
4. Verify the etl succeeds
5. Verify the app still works
